namespaces/live-1.cloud-platform.service.justice.gov.uk/pathfinder-dev/resources/rds.tf

```diff
@@ -59,8 +59,8 @@ data "aws_iam_policy_document" "policy" {
     resources = [
       "arn:aws:rds:${data.aws_region.current.name}:${data.aws_caller_identity.current.account_id}:rds:*",
       "arn:aws:rds:${data.aws_region.current.name}:${data.aws_caller_identity.current.account_id}:snapshot:*",
```

Comment:
Not sure if the region and account id are successfully populated in the final policy which may explain some access denied issues.